src/components/pages/user/list/download-logs/metadata.tsx

```diff
@@ -68,9 +68,13 @@ export const downloadLogsRow = (data, downloadLabel, unknown) => {
               variant="outline"
               size="sm"
               as="a"
-              href={`${ENDPOINT.RAW}/${values.filePath}`}
+              href={
+                values.filePath.startsWith("/naksha")
+                  ? `${API_ENDPOINT}${values.filePath}`
```

Comment:
can you just do use raw w/ `../` instead of exporting `API_ENDPOINT` ? otherwise add new property in ENDPOINT object with `MAIN`

Reply:
since this will be only accessible throgh website we don't have to worry about prefix here since /naksha-api will always be absolute to domain